Tests/package.json

```diff
@@ -2,8 +2,37 @@
   "name": "jsruntimehost",
   "version": "1.0.0",
   "private": true,
+  "scripts": {
+    "build": "webpack",
+    "postinstall": "npm run build"
+  },
   "dependencies": {
-    "chai": "^4.3.4",
-    "mocha": "^9.2.2"
+    "chai": "5.2.0",
+    "jsc-android": "^241213.1.0",
+    "mocha": "11.1.0",
```

Comment:
Using this specific version as more recent ones (at least the one I've tested) errors:
```
ERROR in node:path
Module build failed: UnhandledSchemeError: Reading from "node:path" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.
```